alf/bin/train_play_test.py

```diff
@@ -36,17 +36,28 @@ def run_cmd(cmd, cwd=None):
         cmd (list[str]): command and args to run
         cwd (str): working directory for the process
     """
+
+    def format_error_message(cmd: list, stdout: bytes, stderr: bytes):
+        cmd_inline = ' '.join(cmd)
+        stdout_str = stdout.decode('utf-8')
+        stderr_str = stderr.decode('utf-8')
+        return f'cmd: {cmd_inline} exit abnormally, with\n' \
```

Comment:
The formatting string could be more friendly, e.g., each of 'cmd', 'OUT' and 'ERR' is highlighted in one line.

Reply:
Sure thing. This is fixed.